.travis.yml

```diff
@@ -1,3 +1,9 @@
 language: ruby
 rvm:
   - 2.1.0
+
+branches:
```

Comment:
So what about this? It simply disables all old branches and tags we don't need to take care about even if the switch within the TravisCI ui is not set.

```yml
branches:
  except:
    - old-master
    - release/betty/master
    - release/elefante/master
    - release/essex-hack/master
    - release/fledermaus/master
    - release/fred/master
    - release/hydrogen/master
    - release/pebbles/master
    - release/roxy/master
    - /^release\/.*\/master-bootstrap-ui$/
    - /^release\/hadoop-.*$/
    - /^release\/mesa-.*$/
    - /^release\/suse-cloud\/.*$/
```